docs/configuration/local.md

```diff
@@ -23,9 +23,13 @@ The public docker images are tagged with the same version numbers as the code, w
 
 
 ## Singularity image
-Many HPC environments are not able to run Docker due to security issues. [Singularity](http://singularity.lbl.gov/) is a tool designed to run on such HPC systems which is very similar to Docker. Even better, it can use create images directly from dockerhub.
+Many HPC environments are not able to run Docker due to security issues. [Singularity](http://singularity.lbl.gov/) is a tool designed to run on such HPC systems which is very similar to Docker. There is a particular profile that will download the singularity image for you.
 
-To use the singularity image for a single run, use `-with-singularity 'docker://nfcore/eager'`. This will download the docker container from dockerhub and create a singularity image for you dynamically.
+```bash
+nextflow run nf-core/eager -profile singularity --reads '<path to your reads>' --pairedEnd
+```
+
+Additionally, it can use create images directly from dockerhub. To use the singularity image for a single run, use `-with-singularity 'docker://nfcore/eager'`. This will download the docker container from dockerhub and create a singularity image for you dynamically.
```

Comment:
Is the `-with-singularity` flag out of date, if there is now `-profile singularity`?

Reply:
Its possible to use this even with other profiles, so its still valid :-) It would be possible to use e.g. the base profile with the -with-singularity <path_to_image> switch and run it the same way...